Fluid/Fluid.csproj

```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1;net46</TargetFrameworks>
```

Comment:
At some point might consider dropping the netstandard1.6 altogether, net46(1) should be enough for everyone

Reply:
Yes, I actually did that two days ago on yessql.